libs/sdk-ui-dashboard/src/model/state/config/configSelectors.ts

```diff
@@ -128,6 +128,15 @@ export const selectDisableDefaultDrills = createSelector(selectConfig, (state) =
     return state.disableDefaultDrills ?? false;
 });
 
+/**
+ * Returns whether filter values in drill events should be resolved.
+ *
+ * @alpha
+ */
+export const selectEnableFilterValuesResolutionInDrillEvents = createSelector(selectConfig, (state) => {
+    return state.enableFilterValuesResolutionInDrillEvents;
```

Comment:
Should we provide a default value so this always returns a boolean (similarly to `selectDisableDefaultDrills`)? It could make the results of this easier to use.